src/components/SameBcHierarchyTable/SameBcHierarchyTable.tsx

```diff
@@ -158,17 +155,17 @@ export const SameBcHierarchyTable: FunctionComponent<SameBcHierarchyTableProps>
                 key: item.key,
                 dataIndex: item.key,
                 render: (text: string, dataItem: any) => {
-                    if (item.type === FieldType.multivalue) {
+                    if (item.type === FieldType.multifield || item.type === FieldType.multivalue) {
                         return (
-                            <MultivalueHover
-                                data={(dataItem[item.key] || emptyMultivalue) as MultivalueSingleValue[]}
-                                displayedValue={item.displayedKey && dataItem[item.displayedKey]}
+                            <Field
+                                bcName={bcName}
+                                cursor={dataItem.id}
+                                widgetName={props.meta.name}
+                                widgetFieldMeta={item.type === FieldType.multivalue ? { ...item, type: FieldType.multivalueHover } : item}
```

Comment:
I don't think that arbitrary replacing one field type with another will lead us anywhere.

Seemingly correct way would be simply fix widgets descriptions themselves: find all table widgets, replace all `multivalue` fields with `multivalueHover` fields.   
If for some reason this can't be done it still should be handled on client application side:
```ts
// Custom component for multivalue field
const Multivalue = (props) => {
    const { widgetName } = props
    const widget = useSelector(store => store.view.widgets.find(item => item.name === widgetName))
    if (TableTypes.includes(widget.type)) {
        return <MultivalueHover {...props} />
    }
    return <Multivalue {...props} />
}
```
```ts
// AppLayout
<View
    customFields={{
        [FieldType.multivalue]: Multivalue
    }}
/>
```

Reply:
This method is chosen to keep the current behavior.
Yes, we can do this on the client side (with fix processing multivalue on tables in Tesler UI), but other client applications also expect to see multivalueHover instead of multivalue on tables.